test/unit/postprocessing/test_threshold_optimization.py

```diff
@@ -245,7 +253,8 @@ def test_threshold_optimization_equalized_odds(score_transform, y_transform,
     # returns score_transform(scores_ex) as output
     estimator = ThresholdOptimizer(estimator=PassThroughPredictor(score_transform),
                                    constraints='equalized_odds',
-                                   flip=True)
+                                   flip=True,
+                                   predict_method='predict')
```

Comment:
I suppose we don't have tests for the `'auto'` case yet, right? It worries me a bit to not have a very basic one. For more extensive ones you can always open an issue IMO

Reply:
The `auto` case is tested in `test_common.py`.

Reply:
Yes, but only that "unit", as in `_get_soft_predictions`, but not end-to-end.